AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -314,11 +314,13 @@ public class CardComponent: PaymentComponent, PresentableComponent, Localizable,
     }
     
     private func requestCardTypes(for bin: String) {
-        cardTypeProvider.requestCardTypes(for: bin, supported: self.supportedCardTypes) { [weak self] cardTypes in
+        let parameters = CardBrandProviderParameters(bin: bin, supportedTypes: supportedCardTypes)
```

Comment:
there are only 2 parameters, what is the point of wrapping it in standalone struct?

Reply:
Yeah, tell that to CodeBeat :-D , it was complaining about AnyCardBrandProvider main function having too many parameters :-).